src/libs/actions/Report.ts

```diff
@@ -1027,7 +1027,6 @@ function navigateToAndOpenReportWithAccountIDs(participantAccountIDs: number[])
  */
 function navigateToAndOpenChildReport(childReportID = '0', parentReportAction: Partial<ReportAction> = {}, parentReportID = '0') {
     if (childReportID !== '0') {
-        openReport(childReportID);
```

Comment:
@bernhardoj I am confused why we removed the `openReport` call from here when I have approved the PR then this change was not here?

Reply:
Ah, my bad. It's from my lint commit here: https://github.com/Expensify/App/pull/42158/commits/6d0630d35122717a484469b25e78fad35dbaa825

I think I accidentally committed it when working on this [PR](https://github.com/Expensify/App/pull/43250)

Reply:
Oh, it is from another PR, but it was intentional to remove it, so there is no need to revert this change, right?

Reply:
Yes, it was intentional to remove it.